R/objects.R

```diff
@@ -1525,6 +1585,66 @@ as.Graph.matrix <- function(x, ...) {
   return(as.Graph.Matrix(x = as(object = x, Class = 'Matrix')))
 }
 
+#' @param counts name of the CellDataSet assay to store as \code{counts}
+#' @param data name of the CellDataSet assay to slot as \code{data}
```

Comment:
Parameter `data` is neither defined nor used in the function. Will it be used or can we remove it?